mkdocs.yml

```
@@ -41,6 +41,37 @@ plugins:
           rendering:
             show_root_heading: true
             show_source: false
+  - i18n:
+      default_language: en
+      languages:
+        en:
+          name: English
+          build: true
```

I prefer `build=false` here to avoid duplication
```suggestion
          build: false
```